internal/database/postgres/migration/000001_init-tables.up.sql

```diff
@@ -0,0 +1,49 @@
+CREATE TABLE roles (
+                      name VARCHAR PRIMARY KEY
```

Comment:
why did you removed id field?

Reply:
I thought it would be easier to work with